test/functionalTest/cases/0000_content_related_headers/missing_content_type_header.test

```diff
@@ -70,27 +70,27 @@ orionCurl --url "$url" --payload "$payload" --in "application/json; charset=UTF-
 --REGEXPECT--
 1: ++++++++++++++++++++
 HTTP/1.1 415 Unsupported Media Type
-Content-Length: 161
+Content-Length: 160
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "orionError": {
+    "errorCode": {
```

Comment:
This seems to be a bugfix. Maybe it should be mentioned in CNR, something like:

```
- Fix: right error payload using errorCode (previously orionError was used) in POST /v1/queryContext and POST /v1/updateContext in some cases
```

or something like that.

Reply:
True, this bugfix should be in CNR

Reply:
Fixed in 1e2e62c